schema.sql

```diff
@@ -8,3 +8,5 @@ CREATE TABLE animals (
     neutered BOOLEAN,
     weight_kg DECIMAL
 );
+
+ALTER TABLE animals ADD COLUMN species TEXT;
```

Comment:
- Optional
I would recommend using `varchar` instead of `TEXT` here, since species will always be a short word and `TEXT` is generally used for long sentences.